tox.ini

```diff
@@ -13,6 +13,8 @@ commands =
     zope-testrunner --test-path=src []
 deps =
     .[test]
+setenv =
+    ZOPE_INTERFACE_STRICT_IRO = 1
```

Comment:
Can we also do this on Travis too, please? It could either be added to the buildout configuration, or [directly in .travis.yml](https://github.com/zopefoundation/zope.configuration/blob/ce25c151d08579690c0e75e9cb88909a8d3c6068/.travis.yml#L4-L6).

Reply:
Done in 6dfe525403273139e8ed3fefb559d9b10fe9029a.